ydb/library/yql/minikql/arrow/arrow_util.h

```diff
@@ -39,6 +40,27 @@ inline std::string_view GetStringScalarValue(const arrow::Scalar& scalar) {
     return std::string_view{reinterpret_cast<const char*>(base.value->data()), static_cast<size_t>(base.value->size())};
 }
 
+template<ui8 Value>
```

Comment:
please use a normal parameter here, not a template. it doesn't affect perfomance

Reply:
[Fixed](https://github.com/ydb-platform/ydb/pull/3400/files#diff-d1a86e68d1bedb53992628d8e160cd0d33fd9ee0f67bed1bc5294f807a0dd178R43).